components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -259,40 +228,17 @@ subroutine aero_model_init( pbuf2d, species_class, iflagaa )
     !    (1) All of the "if ( convproc_do_aer .or. convproc_do_gas ) then" statements 
     !        in aero_model.F90 have been changed to "if ( convproc_do_aer ) then"
     !    (2) convproc_do_aer=.false. and convproc_do_gas=.true. is no longer allowed.
-    if ( ( .not. convproc_do_aer ) .and. convproc_do_gas ) then
-       errmes = 'aero_model_init - ' // &
-          'convproc_do_aer MUST BE .true. when convproc_do_gas is .true.' 
+    ! for C++ porting: All convproc_do_aer=.false. and resus_fix=.false. conditions
+    ! are removed as not been used/tested. These conditions are then not allowed.
+    ! convproc_do_gas is not not used
```

Comment:
should there be two `not`s here?

Reply:
We can change this line as: `if (.not. (convproc_do_aer .and. resus_fix)) then`
Do you mean this?

Reply:
I was referring to `! convproc_do_gas is not not used`, thinking maybe it means `! convproc_do_gas is not used`

Reply:
aha, thank you for catching it. I will remove one